lambda/logserver/logserver.go

```diff
@@ -118,7 +121,13 @@ func (ls *LogServer) handler(res http.ResponseWriter, req *http.Request) {
 		switch event.Type {
 		case "platform.start":
 			ls.lastRequestIdLock.Lock()
-			ls.lastRequestId = event.Record.(map[string]interface{})["requestId"].(string)
+			switch event.Record.(type) {
+			case map[string]interface{}:
+				ls.lastRequestId = event.Record.(map[string]interface{})["requestId"].(string)
+			case string:
+				recordString := fmt.Sprintf("%s", event.Record)
```

Comment:
This can probably just be:

```
recordString   :=   string(event.Record)
```

Reply:
@kolanos It looks like `.(string)` is the solution.